boards/arm/frdm_k22f/frdm_k22f_defconfig

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2018 Prevas A/S
+# Copyright (c) 2019 Thomas Burdick <thomas.burdick@gmail.com>
+
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ARM=y
+CONFIG_SOC_MK22F51212=y
+CONFIG_SOC_SERIES_KINETIS_K2X=y
+CONFIG_BOARD_FRDM_K22F=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_GPIO=y
+CONFIG_PINMUX=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=72000000
+CONFIG_OSC_LOW_POWER=y
```

Comment:
Add `CONFIG_ARM_MPU=y`

Reply:
So unfortunately mcuxpresso sdk doesn't seem to actually have any of the MPU definitions setup, so when I enable this SYSMPU in particular isn't defined anywhere

Reply:
I've filed a ticket to look into this
https://github.com/zephyrproject-rtos/zephyr/issues/19152